app/controllers/admin/api/backend_apis/services_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class Admin::Api::BackendApis::ServicesController < Admin::Api::BackendApis::BaseController
+  wrap_parameters Service, include: Service.attribute_names | %w[state_event]
+  representer Service
+
+  # TODO: ActiveDocs
+  # TODO: Refactor
+  # TODO: should it all be called product instead of service?
+  # TODO: paginate
+
+  def index
```

Comment:
This could be useful in https://github.com/3scale/porta/blob/9b79b8b4177e0c9cd0fba99f6084bf9adabc4927/app/controllers/admin/api/backend_apis_controller.rb